test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get root_url
```

Comment:
We had this issue with an earlier test. Looks like rails 5 change the the get(/etc) to take a string path, instead of a symbol.  

get :index will now attempt to get 'http://example.com:80index'  

The get method kicks off the web request and populates the results into the response. It accepts 4 arguments:

```The action of the controller you are requesting. This can be in the form of a string or a route (i.e. articles_url).```  

from around: http://edgeguides.rubyonrails.org/testing.html#available-request-types-for-functional-tests